### ISSUE TITLE
Arrange library items to folders

### ISSUE THREAD

**Issue by alevas** [Stremio/stremio-features#83]
It would be nice if the library, appart from Movies/Series/Channels could be further arranged to folders. Eg put series in folders per theme/age/preferred time you want to watch them etc.

**Comment by LeBromX** [comment]
I support this request.

**Comment by elieberman** [comment]
agreed, library folders, and sort by last episode for shows would all be great adds

**Comment by amalija-ramljak** [comment]
It would be great if this feature was added and pretty logical to have. This + the archive feature request would make it much easier to not have to use other apps to keep track of the lists of tv shows and films. In Stremio, it's just too much of a messy pile right now.

**Comment by Gators25** [comment]
My request 1163 was closed today as a duplicate.

Is this the duplicate from years ago, to where there still isn't folders?

**Comment by ChimeraGaming** [comment]
Nice :) Folder cleanup. If you want a temp solution until stremio gets this going I have a web only extension that adds this feature (not transferrable to apps as it's localized to the pc it is on)

**Comment by Gators25** [comment]
Any update to why no folders?

**Comment by ChimeraGaming** [comment]
Last I heard in Stremio's Discord was they're working on it but it's been a few months. I can't do it unless I want to pay $30+/mo to keep a server going
